src/CMakeLists.txt

```diff
@@ -27,40 +49,105 @@ if(WIN32)
 message(STATUS "Building Iaito version ${IAITO_VERSION_FULL}")
 add_definitions("-DAPP_VERSION=\"${IAITO_VERSION_FULL}\"")
 
-
-add_executable(iaito ${UI_FILES} ${QRC_FILES} ${SOURCE_FILES} ${HEADER_FILES})
-qt5_use_modules(iaito Core Widgets Gui WebEngine WebEngineWidgets)
-target_link_libraries(iaito ${RADARE2_LIBRARIES})
+set(iaito_SRCS
+	${CMAKE_SOURCE_DIR}/analthread.cpp
+	${CMAKE_SOURCE_DIR}/createnewdialog.cpp
+	${CMAKE_SOURCE_DIR}/helpers.cpp
+	${CMAKE_SOURCE_DIR}/hexascii_highlighter.cpp
+	${CMAKE_SOURCE_DIR}/hexhighlighter.cpp
+	${CMAKE_SOURCE_DIR}/highlighter.cpp
+	${CMAKE_SOURCE_DIR}/main.cpp
+	${CMAKE_SOURCE_DIR}/mainwindow.cpp
+	${CMAKE_SOURCE_DIR}/mdhighlighter.cpp
+	${CMAKE_SOURCE_DIR}/newfiledialog.cpp
+	${CMAKE_SOURCE_DIR}/optionsdialog.cpp
+	${CMAKE_SOURCE_DIR}/qrcore.cpp
+	${CMAKE_SOURCE_DIR}/qrdisasm.cpp
+	${CMAKE_SOURCE_DIR}/radarewebserver.cpp
+)
+
+set(iaitoWidgets_SRCS
+	${CMAKE_SOURCE_DIR}/widgets/codegraphic.cpp
+	${CMAKE_SOURCE_DIR}/widgets/commentswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/consolewidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/dashboard.cpp
+	${CMAKE_SOURCE_DIR}/widgets/flagswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/functionswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/importswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/memorywidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/notepad.cpp
+	${CMAKE_SOURCE_DIR}/widgets/omnibar.cpp
+	${CMAKE_SOURCE_DIR}/widgets/pieview.cpp
+	${CMAKE_SOURCE_DIR}/widgets/relocswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/sdbdock.cpp
+	${CMAKE_SOURCE_DIR}/widgets/sectionsdock.cpp
+	${CMAKE_SOURCE_DIR}/widgets/sectionswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/sidebar.cpp
+	${CMAKE_SOURCE_DIR}/widgets/stringswidget.cpp
+	${CMAKE_SOURCE_DIR}/widgets/symbolswidget.cpp
+)
+
+set(iaitoDialogs_SRCS
+	${CMAKE_SOURCE_DIR}/dialogs/aboutdialog.cpp
+	${CMAKE_SOURCE_DIR}/dialogs/commentsdialog.cpp
+	${CMAKE_SOURCE_DIR}/dialogs/renamedialog.cpp
+	${CMAKE_SOURCE_DIR}/dialogs/xrefsdialog.cpp
+)
+
+qt5_wrap_ui(iaito_UI
+	${CMAKE_SOURCE_DIR}/createnewdialog.ui
+	${CMAKE_SOURCE_DIR}/mainwindow.ui
+	${CMAKE_SOURCE_DIR}/newfiledialog.ui
+	${CMAKE_SOURCE_DIR}/optionsdialog.ui
+	${CMAKE_SOURCE_DIR}/dialogs/aboutdialog.ui
+	${CMAKE_SOURCE_DIR}/dialogs/commentsdialog.ui
+	${CMAKE_SOURCE_DIR}/dialogs/renamedialog.ui
+	${CMAKE_SOURCE_DIR}/dialogs/xrefsdialog.ui
+	${CMAKE_SOURCE_DIR}/widgets/commentswidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/consolewidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/dashboard.ui
+	${CMAKE_SOURCE_DIR}/widgets/flagswidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/functionswidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/importswidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/memorywidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/notepad.ui
+	${CMAKE_SOURCE_DIR}/widgets/relocswidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/sdbdock.ui
+	${CMAKE_SOURCE_DIR}/widgets/sectionsdock.ui
+	${CMAKE_SOURCE_DIR}/widgets/sidebar.ui
+	${CMAKE_SOURCE_DIR}/widgets/stringswidget.ui
+	${CMAKE_SOURCE_DIR}/widgets/symbolswidget.ui
+)
```

Comment:
I agree with you about leaving the part which parses Qt project file for Iaito; considering this is in development and likely to have lot of changes, I will flip it back in cmake file.